cves/kernel/CVE-2013-3223.yml

```diff
@@ -105,10 +120,13 @@ vcc_instructions: |
 
   Place any notes you would like to make in the notes field.
 vccs:
-- commit: "1da177e4c3f41524e886b7f1b8a0c1fc7321cac2"
-  note: Discovered automatically by archeogit.
-- commit: 98e399f82ab3a6d863d1d4a7ea48925cc91c830e
-  note: Discovered automatically by archeogit.
+- commit: 940649fbecb9748286d0f65d57395aaea79b8e74
+  note: |
+    The bug seems to exist every since the AX.25 socket driver was introduced
```

Comment:
this is just a small thing -- but theres just a typo about the "ever" instead of every